src/consumers/index.js

```diff
@@ -32,6 +33,7 @@ const consumers = [
   createToken,
   createTransformedERC20EventFills,
   createUniswapV2SwapEventFill,
+  createUniswapV3SwapEventFill,
```

Comment:
This consumer won't be needed as all fill creation is handled through the `createFillsForEvent` consumer now. The existing `createUniswapV2SwapEventFill` consumer and some others need removing at some point 😁